com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -648,6 +666,7 @@ public void SetConnectionData(NetworkEndpoint endPoint, NetworkEndpoint listenEn
             SetConnectionData(serverAddress, endPoint.Port, listenAddress);
         }
 
+#if !UTP_TRANSPORT_2_0_ABOVE
```

Comment:
If only deprecating the debug simulator, should conditionally add an obsolete argument here.

Reply:
I'm only deprecating the `DebugSimulator` if we use `UTP_TRANSPORT_2_0_ABOVE` in this case, for UTP1, it's still valid.

Reply:
Right, my point is that this piece of code is compiling `SetDebugSimulatorParameters` out entirely if UTP 2.0 is installed. If the intention is to only deprecate it, then we should do as above and conditionally compile the `Obsolete` attribute, and not the entire method.

Reply:
I see, that makes sense, I've updated the code.